docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1037,100 @@ def generate_all_docs(self, readme_content: str):
 
         logger.info("Plugin documentation generated successfully!")
 
 
+class LabsDocGenerator(PluginDocGenerator):
+    """Generates Labs documentation from the fiftyone-labs repository."""
+
+    _ecosystem_subdir = "labs_ecosystem"
+    _docs_section = "labs"
+    _card_link_prefix = "../labs/"
+
+    def __init__(self, docs_source_dir: str):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)